src/pages/patientView/trialMatch/TrialMatchTable.tsx

```diff
@@ -196,6 +196,12 @@ export default class TrialMatchTable extends React.Component<ITrialMatchProps> {
         );
     }
 
+    public openFeedbackForm(nctId: string, protocolNo: string) {
+        this.showTrialFeedback = true;
```

Comment:
instead of having a flag, you can just can just have selectedTrialFeedbackFormData which is either undefined (hide) or shown (populated).  In general, it's best not to update multiple properties like this.

Reply:
Removed the flag